### ISSUE TITLE
We have an API that uses a `locale` property

### ISSUE THREAD

**Issue by les2** [esbanarango/ember-model-validator#65]
There are a bunch of pre-existing code that uses the `locale` property of the model:

```
// my-model
export default Model.extend(Validator, {
  locale: attr('string'),
  ...
})
```

This conflicts with the locale property defined here: https://github.com/esbanarango/ember-model-validator/blob/master/addon/mixins/model-validator.js#L31-L38

There are obvious workarounds:
* monkey-patch `Validator`
* use `myLocale: attr('string', { key: 'locale' })` and *update all existing code*

In general, Ember addons tend to be very aggressive with claiming keys in users' namespaces (see https://github.com/emberjs/data/issues/973), so this code is idiomatic.

But what's the suggested workaround?

Also posting so that anyone else with the problem can find it (and a solution, if there is one).

cc @rwwagner90 @jblakeman

**Comment by nojacko** [comment]
I've just come across the same issue. :(

**Comment by esbanarango** [comment]
Fixed by #107
